semantic/check.go

```diff
@@ -7,6 +7,10 @@ func runChecks(pkg *Package) error {
 	if err := checkOptionDecs(pkg); err != nil {
```

Comment:
I had to do a double take to figure out that  `checkOptionDecs` and  `checkOptionDeps` are different. Maybe we should rename `Decs` to Assignments? Something like `checkOptionAsgns`?

Reply:
I agree. Thanks for the catch.